dist/tools/fixdep/Makefile

```diff
@@ -0,0 +1,11 @@
+all: fixdep
+
+fixdep_patched.c: fixdep.c
+	patch $^ fixdep_riot.patch -o $@
+
+fixdep: fixdep_patched.c
+	$(CC) -O3 -Wall $^ -o $@
```

Comment:
I think this will try to cross-compile the tool. Maybe do something like edbg?

https://github.com/RIOT-OS/RIOT/blob/16f47fc8932ccf3de7a5775944c22f6943cc9e78/dist/tools/edbg/Makefile#L17-L22